website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -138,6 +138,8 @@ data:
   ...
 ```
 
+For Spot Interruption Warnings, Karpenter will cordon and drain the node as per the above. There may be insufficient capacity to schedule the pods running on the affected node. The provisioner will then start a new instance to schedule these pods. Spot Interruptions have a 2 minute notice before Amazon EC2 reclaims the instance. Generally, this is sufficient time for the new node to become ready and to schedule the pods. Karpenter will follow the PDB while it drains the node. This means there is a chance that a node may not be fully drained by the time it is reclaimed.
```

Comment:
Can you add this detail higher up where we say "When Karpenter detects..." at the end. I'm imagining the key details that we need to include here beyond what's currently there is that:
1. Karpenter spins up a new instance as soon as it sees the spot interruption
2. The spot interruption comes with a 2-minute warning, which is effectively the time that Karpenter has to spin up the new node
3. Mention that Karpenter's average node startup time means that this 2 minute startup is generally enough to spin up the new node and transfer the pods over.

Reply:
Thanks @jonathan-innis  ! Addressed in latest commit